### PR TITLE
Added lesson04

### PR DESCRIPTION
**Pull request by Chieu1**
Added dict_lab.py and mailroom_part2.py on lesson04

### CODE REVIEW

File: students/chieu_quach/lesson03/mailroom.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # Assignment -  Lesson 3
 # Exercise   -  Mailroom Part 1
 
+# mailroom part 1
```
Comment (charlesrobison) [inline students/chieu_quach/lesson03/mailroom.py]: @Chieu1 - I'm unable to do inline comments on this file due to the issues with submitting this for review.  But, overall it works well.  There are a few PEP8 issues you'll want to address, like extra blank lines.  Take a look at installing a linter on you text editor to catch those.  Also, your import statements should all be at the top of the file.

File: students/chieu_quach/lesson04/dict_lab.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+
+# Author      :  Chieu Quach
+# Assignment  :  Lesson 4
+# Exercise    :  Dictionary and Set Lab
+
+# first variable = key : second variable = value
+# Example = "Name" is key : "Chris" is value
+
+def dictionaries():
+
```
Comment (charlesrobison) [inline students/chieu_quach/lesson04/dict_lab.py]: Remove these blank lines

File: students/chieu_quach/lesson04/mailroom_part2.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+       response = input(prompt)
+       response = int(response)
+       switch[response]()
+       
+     
+
+# Main function
+if __name__ == "__main__": 
+  
+   main()
```
Comment (charlesrobison) [inline students/chieu_quach/lesson04/mailroom_part2.py]: @Chieu1 - this version isn't working as expected.  When sending a thank you to a single donor, if I enter a name, it breaks out of the program.  Also, the program should continue to loop until the user exits.  Also, there are a lot of PEP8 issues - mainly spacing.  All indentations should be 4 spaces in Python.

**Comment by Chieu1** [comment]
Charles,

I will try to install Linter for text editor.
As you mentioned on the mailroom for single donor, it will work if you enter full name. I will fix it so you can just emter the first name and it will work. I will create a loop  to the menu.

Thanks,

Chieu

**Pull request merged**